### ISSUE TITLE
Incompatible with Recalbox 6.0 Dragonblaze ?

### ISSUE THREAD

**Issue by fmarot** [nordicpower/gamelistpower#3]
Hello, I've tried to install "gamelistpower" inside the latest Recalbox 6.0 Dragonblaze but it seems not to work.
I've tried twice by reinstalling on a blanck SD card and it's the same each time: no new entry in the sytems appear.
From what I see in the "es_systems.cfg" there is a new entry but not being considered by emulation station. In the logs I can see:

[2019/05/14 00:16:07.049] (WARN!) : Key: collectionsdreamcast
[...]
[2019/05/14 00:16:07.085] (INFO ) : creating thread for system collections
[...]
[2019/05/14 00:16:19.942] (ERROR) : System "collections" has raised an error. Ignored.

Do you have an idea ?

**Comment by fmarot** [comment]
One more detail:

When I modify es_system.cfg to have the following content, then the "collections" games and scripts work correctly but no other game from any system launches ! They all crash on startup and back to emulationstation.

```
<system>
		<fullname>Amiga 600</fullname>
		<name>amiga600</name>
		<path>/recalbox/share/roms/collections</path>
		<extension>.sh</extension>
		<command>%ROM% %CONTROLLERSCONFIG% %RATIO%</command>
		<platform>amiga</platform>
		<theme>amiga600</theme>
		<emulators />
</system>
```

**Comment by Kintao** [comment]
You have a blank here : 
`<emulators />`
Replace it with : 
`<emulators/>`

**Comment by spooner59** [comment]
I have the same problem under recalbox 6.0 nothing appears in the menu, someone can help me?

**Comment by fmarot** [comment]
> You have a blank here :
> `<emulators />`
> Replace it with :
> `<emulators/>`

Thanks @Kintao but I'm sorry:
1- I will not be able to test anytime soon
2- are you sure ? I think that "<emulators/>" is valid xml so it 'should' not be a problem

Nevertheless, thanks for your answer

**Comment by bozothegeek** [comment]
> Hello, I've tried to install "gamelistpower" inside the latest Recalbox 6.0 Dragonblaze but it seems not to work.
> I've tried twice by reinstalling on a blanck SD card and it's the same each time: no new entry in the sytems appear.
> From what I see in the "es_systems.cfg" there is a new entry but not being considered by emulation station. In the logs I can see:
> 
> [2019/05/14 00:16:07.049] (WARN!) : Key: collectionsdreamcast
> [...]
> [2019/05/14 00:16:07.085] (INFO ) : creating thread for system collections
> [...]
> [2019/05/14 00:16:19.942] (ERROR) : System "collections" has raised an error. Ignored.
> 
> Do you have an idea ?

Yes, there is a missing tag in es_systems.cfg : <emulators></emulators> and there is a problem to execute .sh in my 6.1.1, I'm working on it at the moment...
